client/dashboard/me/security-account-recovery/index.tsx

```diff
@@ -17,6 +18,7 @@ export default function SecurityAccountRecovery() {
 				/>
 			}
 		>
+			<ComponentViewTracker eventName="calypso_dashboard_security_account_recovery_page_view" />
```

Comment:
I don't think all these new `_page_view` events are the right. For tracking page views we already have the standard `calypso_page_view` event which we use everywhere for tracking standard page views.

See https://github.com/Automattic/wp-calypso/blob/a0e8b9cdb22c84c413af9dea584bc474ea66fb43/client/dashboard/app/root/index.tsx#L106

The `<ComponentViewTracker>` tracker is for tracking upsell or banner impressions. Things that appear conditionally.